openshift/jenkins-proxy.app.yaml

```diff
@@ -89,6 +89,8 @@ objects:
               secretKeyRef:
                 name: jenkins-proxy
                 key: postgres.password
+          - name: JC_DEBUG
```

Comment:
What does JC stand for?

Reply:
When I started to write idler I called it Jenkins Controller, so I used prefix `JC`...would be probably a good thing to refactor in the future so that it makes more sense..

Reply:
+1

Reply:
I created issue #87